packages/headless/src/features/configuration/configuration-actions.ts

```diff
@@ -27,31 +28,55 @@ export interface UpdateBasicConfigurationActionCreatorPayload {
   accessToken?: string;
 
   /**
-   * The unique identifier of the target Coveo Cloud organization (e.g., `mycoveocloudorganizationg8tp8wu3`)
+   * The environment in which the organization is hosted.
+   *
+   * The `dev` and `stg` environments are only available internally for Coveo employees (e.g., Professional Services).
```

Comment:
- [ ] maybe `@internal`

Reply:
I'm not sure hiding them from the generated documentation is any better: the `dev` and `stg` values will be visible through API discovery, e.g., 

<img width="825" alt="Screenshot 2024-09-17 at 4 14 48 PM" src="https://github.com/user-attachments/assets/434bdf55-f1ab-4f6e-b890-dda9b57e6215">

so hiding them could actually end up being more confusing.

@jpmarceau what do you think?

Regardless, I don't think we should block merging this PR because of that (we can apply the comment later) so I'll resolve this comment.

Reply:
Marking the whole `environment` parameter `internal`, if doable, would probably be ideal, because there's no need for clients to even know that the param exists, let alone the `dev` and `stg` values.
But if not, then yeah, just having a helpful comment like Fred wrote sounds good to me.
And either way, I agree with Fred that this shouldn't hold up the release 🚢